File: C/Exercicio36.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  
  int numeros[20], multi, resul, i;
  
  printf("Digite os 20 numeros: ");
  for(i=1; i<=20; i++){
    scanf("%i", &numeros[i]);
  }
  
  printf("Digite o valor da constante multiplicadora: ");
  scanf("%i", &multi);

  for(i=1; i<=20; ++i){
    resul = (numeros[i] * multi);
    printf("\nO valor de %i vezes %i e: %i", numeros[i], multi, resul);    
  }

  return 0;
}

File: Python/Exercicio36.py
numeros = []
vetor2 = []

for i in range(0, 20, 1):
    x = int(input('Digite um numero: '))
    numeros.append(x)

mult = int(input("Digite a constante multiplicativa: "))

for i in range(0, 20, 1):
    print("O valor de", numeros[i], "vezes", mult, "é:")
    numeros[i] = numeros[i]*mult
    vetor2.append(numeros[i])
    print(vetor2[i])